pkg/crdpuller/discovery.go

```diff
@@ -224,8 +226,9 @@ func (sp *schemaPuller) PullCRDs(context context.Context, resourceNames ...strin
 				scaleSubResource = nil
 			}
 
+			hasAdditonalPrinterColumn := len(additionalPrinterColumns) != 0
```

Comment:
Is the `hasAdditonalPrinterColumn` variable really useful here? Can't it be inlined since it is used only once ?`

Reply:
Make sense, thanks point it out @davidfestal